business_rules/operators.py

```diff
@@ -782,6 +782,27 @@ def detect_reference(self, row, value_column, target_column, context=None):
         value = row[value_column]
         return (value in target_data) or (value in target_data.astype(int).astype(str)) or (value in target_data.astype(str))
 
+    @type_operator(FIELD_DATAFRAME)
+    def additional_columns_empty(self, other_value: dict):
+        """
+        The dataframe column might have some additional columns.
+        Example:
+            column - TSVAL
+            additional columns - TSVAL1, TSVAL2, ...
+        So, if these columns exist, they cannot be empty. Original column can be empty.
+        The operator extracts these additional columns from the DF
+        and ensures they are not empty.
+        """
+        target: str = self.replace_prefix(other_value.get("target"))
+        # regexp means starting from target, ending with integers and nothing is between them
+        df = self.value.filter(regex=rf"^{target}\d+$")
+        return df.isnull().any(axis=1)
```

Comment:
I believe this should take into account if the next column is not null 

ex. There should only be an error if TSVAL2 is not null and TSVAL1 is